### ISSUE TITLE
Internet Explorer table rows blow up

### ISSUE THREAD

**Issue by martk70** [dokufreaks/plugin-pagelist#92]
I use Pagelist as a helper for Task and Tag plugin. My table row heights are too much in Internet Explorer, while in Chrome, and FF they are ok. I use the latest dokuwiki (Release 2017-02-19a "Frusterick Manners") and IE should be recent too (as in picture below). All the best and thank you. Martin
![clipboard](https://cloud.githubusercontent.com/assets/13997873/23507127/d60eee8a-ff4b-11e6-85a9-2bb37f3acca9.jpg)
![afbeelding](https://cloud.githubusercontent.com/assets/13997873/23507208/2f137442-ff4c-11e6-879a-f74a75d9c5c3.png)

**Comment by LarsGit223** [comment]
Hi,
thanks for reporting the issue. I got some questions:
1. Is this also happening with lists created with the pagelist plugin or does it only happen with task lists?
1. Did you have this problem before or is it a new problem?
1. If it's new: what happend? Did you update any plugins or changed anyhting else?

Kind Regards,
Lars

**Comment by martk70** [comment]
Hi Lars,



1. I tested it and it also happens with a 'pure' pagelist list.

2. I reported this some months ago, I think at the tag plugin and the template I am using: bootstrap3.

3. I never used the task plugin before and cannot say exactly what has triggered this.



But I encountered it as a problem in an intranet situation, and tested it tonight at quite another server but in the same dokuwiki configuration/template, and pagelist is acting identical to the other configuration (private ubuntu server). Normal in Firefox, as reported in IE.



Thank you for your interest, and good luck. If I might be of any help, please let me know.



Martin van der Knijff, Gouda

http://www.mvak.nl

0182-537217






----- Original Message ----

From: LarsGit223 

To: dokufreaks/plugin-pagelist 

Cc: martk70 , Author 

Sent: Don, 02 Mrt 2017 17:49

Subject: Re: [dokufreaks/plugin-pagelist] Internet Explorer table rows blow up (#92)




Hi,

thanks for reporting the issue. I got some questions:


	Is this also happening with lists created with the pagelist plugin or does it only happen with task lists?
	Did you have this problem before or is it a new problem?
	If it's new: what happend? Did you update any plugins or changed anyhting else?


Kind Regards,

Lars

**Comment by LarsGit223** [comment]
Thanks for your answers Martin.
You mentioned Bootstrap3: does it look different if you use the standard dokuwiki template?

**Comment by martk70** [comment]
Yes, good question:

Turned on the standard template, and there the table rows have normal heights, in IE....

So I should knock on the door of the bootstrap3 template builder?

His template is quite versatile....  Could you discuss this with him with some expert knowledge on these things, or should I contact him?



Thanks again.

Martin





Martin van der Knijff, Gouda

http://www.mvak.nl

0182-537217




----- Original Message ----

From: LarsGit223 

To: dokufreaks/plugin-pagelist 

Cc: martk70 , Author 

Sent: Don, 02 Mrt 2017 21:04

Subject: Re: [dokufreaks/plugin-pagelist] Internet Explorer table rows blow up (#92)




Thanks for your answers Martin.

You mentioned Bootstrap3: does it look different if you use the standard dokuwiki template?

**Comment by LarsGit223** [comment]
Yes, I suggest that.

If I got time for it I might have a look at it, this can take some time. Also I am not a CSS expert and as you mentioned the bootstrap3 template is quite complex.

**Comment by LarsGit223** [comment]
I did a little test with Chrome. The task list looks OK.

But there are problems with the publish plugin. The message generated by the publish plugin appears in every table row of the task list overview. So I disabled it and then the task list looks fine..

**Comment by martk70** [comment]
I changed an issue already reported here:

https://github.com/LotarProject/dokuwiki-template-bootstrap3/issues/278



The developer lists plugins that integrate with his template, so I think pagelist is not yet 'opted-in'. I hope this will help to change this.

https://www.dokuwiki.org/template:bootstrap3



As for the publish plugin, I'll leave this out.



Thank you.

Martin






----- Original Message ----

From: LarsGit223 

To: dokufreaks/plugin-pagelist 

Cc: martk70 , Author 

Sent: Don, 02 Mrt 2017 21:58

Subject: Re: [dokufreaks/plugin-pagelist] Internet Explorer table rows blow up (#92)




I did a little test with Chrome. The task list looks OK.

But there are problems with the publish plugin. The message generated by the publish plugin appears in every table row of the task list overview. So I disabled it and then the task list looks fine..
